instance/tests/models/test_openedx_appserver.py

```diff
@@ -201,6 +201,18 @@ def test_launch_in_other_region(self, mock_get_nova_client):  # pylint: disable=
         make_test_appserver(instance)
         mock_get_nova_client.assert_called_once_with("elsewhere")
 
+    def test_default_discovery_version(self):
```

Comment:
I don't see tests for any of the other ``*_VERSION`` variables. So can you make this more extensive so it checks all configured versions using ddt?

Reply:
Added tests for all the `*_VERSION` variables in c3b1abe2a2944ebfd2301d1b69eec472d7bd301c